selfdrive/car/ford/carcontroller.py

```diff
@@ -89,8 +166,21 @@ def update(self, CC, CS, now_nanos):
       gas = accel
       if not CC.longActive or gas < CarControllerParams.MIN_GAS:
         gas = CarControllerParams.INACTIVE_GAS
+
       stopping = CC.actuators.longControlState == LongCtrlState.stopping
-      can_sends.append(fordcan.create_acc_msg(self.packer, self.CAN, CC.longActive, gas, accel, stopping, v_ego_kph=40 * CV.MPH_TO_KPH))
+
+      # Calculate targetSpeed
+      targetSpeed = actuators.speed
+      if not CC.longActive and hud_control.setSpeed:
+        targetSpeed = hud_control.setSpeed
+
+      precharge_actuate, brake_actuate = actuators_calc(self, accel)
```

Comment:
With current values:
For ICE:
- `precharge_actuate` is activated at -0.2 m/s^2 and released at 0 - or after 2 seconds if brake was not activated in the meantime
- `brake_actuate` is activated at -0.3 m/s^2 and released at 0 

For EV:
- `precharge_actuate` and `brake_actuate` are activated at -0.3 m/s^2 and released at 0